test/e2e/fundingManager/BondingCurveTokenRescueE2E.t.sol

```diff
@@ -0,0 +1,257 @@
+        // NOTE: It's important to store the module configurations in order, since _create_E2E_Orchestrator() will copy from the array.
+        // The order should be:
+        //      moduleConfigurations[0]  => FundingManager
+        //      moduleConfigurations[1]  => Authorizer
+        //      moduleConfigurations[2]  => PaymentProcessor
+        //      moduleConfigurations[3:] => Additional Logic Modules
+
+        // FundingManager
+        setUpBancorVirtualSupplyBondingCurveFundingManager();
+
+        // BancorFormula 'formula' is instantiated in the E2EModuleRegistry
+
+        IBondingCurveBase_v1.IssuanceToken memory issuanceToken_properties =
+        IBondingCurveBase_v1.IssuanceToken({
+            name: "Bonding Curve Token",
+            symbol: "BCT",
+            decimals: 18,
+            maxSupply: type(uint).max - 1
+        });
+
+        issuanceToken = new ERC20Issuance_v1(
+            "Bonding Curve Token", "BCT", 18, type(uint).max - 1, address(this)
+        );
+
+        bc_properties = IFM_BC_Bancor_Redeeming_VirtualSupply_v1
+            .BondingCurveProperties({
+            formula: address(formula),
+            reserveRatioForBuying: 333_333,
+            reserveRatioForSelling: 333_333,
+            buyFee: 0,
+            sellFee: 0,
+            buyIsOpen: true,
+            sellIsOpen: true,
+            initialIssuanceSupply: 10,
+            initialCollateralSupply: 30
+        });
+
+        moduleConfigurations.push(
+            IOrchestratorFactory_v1.ModuleConfig(
+                bancorVirtualSupplyBondingCurveFundingManagerMetadata,
+                abi.encode(address(issuanceToken), bc_properties, token)
+            )
+        );
+
+        // Authorizer
+        setUpRoleAuthorizer();
+        moduleConfigurations.push(
+            IOrchestratorFactory_v1.ModuleConfig(
+                roleAuthorizerMetadata, abi.encode(address(this))
+            )
+        );
+
+        // PaymentProcessor
+        setUpSimplePaymentProcessor();
+        moduleConfigurations.push(
+            IOrchestratorFactory_v1.ModuleConfig(
+                simplePaymentProcessorMetadata, bytes("")
+            )
+        );
+    }
+
+    function test_e2e_TokenRescue() public {
+        // address(this) creates a new orchestrator.
+        IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig =
+        IOrchestratorFactory_v1.WorkflowConfig({
+            independentUpdates: false,
+            independentUpdateAdmin: address(0)
+        });
+
+        IOrchestrator_v1 orchestrator =
+            _create_E2E_Orchestrator(workflowConfig, moduleConfigurations);
+
+        FM_BC_Bancor_Redeeming_VirtualSupply_v1 fundingManager =
+        FM_BC_Bancor_Redeeming_VirtualSupply_v1(
+            address(orchestrator.fundingManager())
+        );
+
+        issuanceToken.setMinter(address(fundingManager), true);
+
+        // IMPORTANT
```

Comment:
This section can be removed, outdated

Reply:
You mean the section about how we need a minimum amount of tokens in the contract to start?